bookkeeper-server/src/main/java/org/apache/bookkeeper/client/PendingReadOp.java

```diff
@@ -389,8 +389,10 @@ private void submitCallback(int code) {
                     break;
                 }
             }
-            LOG.error("Read of ledger entry failed: L{} E{}-E{}, Heard from {}. First unread entry is {}",
-                    new Object[] { lh.getId(), startEntryId, endEntryId, heardFromHosts, firstUnread });
+            if (LOG.isDebugEnabled()) {
+                LOG.debug("Read of ledger entry failed: L{} E{}-E{}, Heard from {}. First unread entry is {}",
+                        new Object[] { lh.getId(), startEntryId, endEntryId, heardFromHosts, firstUnread });
+            }
```

Comment:
Any rational why we are converting a read error into a debug message?
Read errors are very important to track.